nixos/modules/services/hardware/cdemu.nix

```diff
@@ -0,0 +1,49 @@
+{ config, lib, pkgs, ... }:
```

Comment:
I would call this file under `programs/cdemu.nix`. It's not a service, it's a series of config to make a program work.